app/src/main/java/nl/tudelft/cs4160/identitychain/Util/Key.java

```diff
@@ -184,15 +192,31 @@ public static PublicKey loadPublicKey(String key) {
         return null;
     }
 
+    public static PublicKey loadByteKey(byte[] rawKey) {
+        KeyFactory kf = getKeyFactory();
+        if (kf == null) {
+            return null;
+        }
+        X509EncodedKeySpec pubKeySpec = new X509EncodedKeySpec(rawKey);
+
+        try {
+            return kf.generatePublic(pubKeySpec);
+        } catch (InvalidKeySpecException e) {
+            e.printStackTrace();
```

Comment:
Yes, I just needed this overload to avoid an android dependency for base64 encoding.
they were taking bytes encodignt o base 64 and then back to bytes... 
and the base 64 was an android dependency. furthermore the exception was never properly logged in the other overload either. I left this as java since I wanted to modify this file as less as possible